packages/website/styles/global.css

```diff
@@ -21,6 +21,13 @@
   font-family: ChicagoFLF, sans-serif;
 }
 
+h2 {
+  font-size: 2em;
```

Comment:
Probably a good idea to move these font sizes into CSS variables, so we can make utility classes for non header elements that need same font size.